### ISSUE TITLE
About loss curve

### ISSUE THREAD

**Issue by quankainwafu** [IvoryCandy/super-resolution#24]
Thank you for your work.

I have tried to train the code and got some good result.I need the loss curve now ,But i am not familiar with pytorch,how to draw the loss curve?

Thank you for reading my question.
